### PR TITLE
Add Ruifeng Zheng to committer list

### PR DESCRIPTION
**Pull request by zhengruifeng**

### CODE REVIEW

**Comment by wangyum** [comment]
Congratulations

**Review by HyukjinKwon — APPROVED** [review]

File: committers.md [hunk 1]
```diff
@@ -78,6 +78,7 @@ navigation:
 |Takeshi Yamamuro|NTT|
 |Burak Yavuz|Databricks|
 |Matei Zaharia|Databricks, Stanford|
+|Ruifeng Zheng|JD.COM|
```
Comment (wangyum) [inline committers.md]: `JD.COM` -> `JD.com`?
https://en.wikipedia.org/wiki/List_of_largest_Internet_companies#List

**Comment by zhengruifeng** [comment]
still can not merge this PR, will try later

**Comment by zhengruifeng** [comment]
Finally I merged this PR, thank @wangyum for your guide!

**Comment by dongjoon-hyun** [comment]
Congratulation, @zhengruifeng ! 😄

**Comment by zhengruifeng** [comment]
Thank you @dongjoon-hyun ! Thanks all!

**Pull request closed without merging**